plugins/org.locationtech.udig.libs/.classpath

```diff
@@ -171,7 +170,7 @@
 	<classpathentry exported="true" kind="lib" path="lib/jfreechart-1.0.13.jar"/>
 	<classpathentry exported="true" kind="lib" path="lib/jfreechart-experimental-1.0.13.jar"/>
 	<classpathentry exported="true" kind="lib" path="lib/jfreechart-swt-1.0.13.jar"/>
-	<classpathentry exported="true" kind="lib" path="lib/jgridshift-1.1.jar"/>
+	<classpathentry exported="true" kind="lib" path="lib/jgridshift-core-1.2.jar"/>
 	<classpathentry exported="true" kind="lib" path="lib/jgt-oms3-0.7.6.jar"/>

```

Comment:
Well, IMHO I would't add this file (in addition to .settings/ folders) to source-repository anymore. M2e plugins within Eclipse IDE would created it from MF-Files /pom.xml files of project. I recently used it and got rid of this eclipse-ide-specific files.

Reply:
Maybe we should accept this PR and then you create a seperate one addressing removal of all .classpath files and .settings folder

Reply:
Was just waiting for the travis build feedback ;)

Thanks for the fix!